components/settings/EditPassword.jsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+const EditPassword = () => {
+  return (
+    <form>
+      <div>
+        <p>Old Password:</p>
+        <input type="password"></input>
+      </div>
+      <div>
+        <p>New Password:</p>
+        <input type="password"></input>
+      </div>
+      <div>
+        <p>Confirm New Password:</p>
+        <input type="password"></input>
+      </div>
+      <button>Save</button>
+    </form>
+  );
+};
+
+export default EditPassword;
```

Comment:
For now the only way a user change change his/her password is through the reset password form.

Reply:
Damola and Vincent created an endpoint for this.